src/proxy-page/proxy-page.service.ts

```diff
@@ -101,6 +102,7 @@ export class ProxyPageService {
     } else if (type !== 'notitle') {
       addHeaderTitle()(this.context);
     }
+    fixSVG(this.config)(this.context)
```

Comment:
We have a different configuration in lint or prettier, because all the lines have semicolon and yours not. 
Linter is showing an error in my local configuration for this line. We need to find the right setting to get the same controls and enforce semicolon, like it was initially.

Reply:
I made some changes in the linter configuration so we should get now more control on the format.